OADP-3787: IBM P & Z platform known issues (#74157)
openshift/openshift-docs: modules/oadp-ibm-p-and-z-known-issues.adoc

// Module included in the following assemblies:
//
// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc

:_mod-docs-content-type: CONCEPT
[id="oadp-ibm-p-and-z-known-issues_{context}"]
= Known issue of OADP using {ibm-power-name} and {ibm-z-name} platforms

> .Backup method limitation for Single Node OpenShift clusters on {ibm-power-name} and {ibm-z-name} platforms

Author: ```suggestion
.Backup method limitation for {sno-caps} clusters on {ibm-power-name} and {ibm-z-name} platforms
```

> The following are backup method restrictions for Single Node Openshift (SNO) clusters deployed on {ibm-power-name} and {ibm-z-name} platforms:

Author: ```suggestion
The following are backup method restrictions for {sno-caps} clusters deployed on {ibm-power-name} and {ibm-z-name} platforms:
```

> Storage requirement:: Only NFS storage is currently compatible with SNO clusters deployed on {ibm-power-name} and {ibm-z-name} platforms.

Author: ```suggestion
Storage requirement:: Only NFS storage is currently compatible with {sno} clusters deployed on {ibm-power-name} and {ibm-z-name} platforms.
```

> Backup method:: Only the File System Backup (kopia, restic) method is supported for backup and restore operations.

Author: ```suggestion
Backup method:: Only the File System Backup (`kopia` and `restic`) method is supported for backup and restore operations.
```

> Backup method:: Only the File System Backup (kopia, restic) method is supported for backup and restore operations.

Reviewer: ```suggestion
Backup method:: Only the File System Backup method such as `kopia` and `restic` are supported for backup and restore operations.
```

> Backup method:: Only the File System Backup ( kopia, restic) method is supported for backup and restore operations.

Author: ```suggestion
Backup method:: Only the File System Backup (kopia, restic) method is supported for backup and restore operations.
```

// Module included in the following assemblies:
//
// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc

:_mod-docs-content-type: CONCEPT
[id="oadp-ibm-p-and-z-known-issues_{context}"]
= Known issue of OADP using {ibm-power-name} and {ibm-z-name} platforms

> 
> .Backup method limitation for {sno-caps} clusters on {ibm-power-name} and {ibm-z-name} platforms
> 
> The following are backup method restrictions for {sno-caps} clusters deployed on {ibm-power-name} and {ibm-z-name} platforms:
> 
> Storage requirement:: Only NFS storage is currently compatible with {sno-caps} clusters deployed on {ibm-power-name} and {ibm-z-name} platforms.
> 
> Backup method:: Only the File System Backup method such as `kopia` and `restic` are supported for backup and restore operations.

Reviewer 2: ```suggestion
* Currently, there are backup method restrictions for Single-node OpenShift clusters deployed on IBM Power® and IBM Z® platforms. Only NFS storage is currently compatible with Single-node OpenShift clusters on these platforms. In addition, only the File System Backup methods such as kopia and restic are supported for backup and restore operations. There is currently no workaround for this issue.
```

Reviewer 2: Replace product names with attributes.
SSG: https://redhat-documentation.github.io/supplementary-style-guide/#conversational-style:~:text=Currently%2C%20you%20cannot%20use%20Orchestration%20(heat)%20templates%20with%20the%20director%20to%20deploy%20an%20overcloud%20that%20requires%20NFS%20as%20an%20Image%20service%20(glance)%20back%20end.%20There%20is%20currently%20no%20workaround%20for%20this%20issue.

// Module included in the following assemblies:
//
// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc

:_mod-docs-content-type: CONCEPT
[id="oadp-ibm-p-and-z-known-issues_{context}"]
= Known issue of OADP using {ibm-power-name} and {ibm-z-name} platforms

> * Currently, there are backup method restrictions for Single-node OpenShift clusters deployed on IBM Power® and IBM Z® platforms. Only NFS storage is currently compatible with Single-node OpenShift clusters on these platforms. In addition, only the File System Backup methods such as kopia and restic are supported for backup and restore operations. There is currently no workaround for this issue.

Author: ```suggestion
* Currently, there are backup method restrictions for {sno-caps} clusters deployed on {ibm-power-name} and {ibm-z-name} platforms. Only NFS storage is currently compatible with {sno-caps} clusters on these platforms. In addition, only the File System Backup (FSB) methods such as Kopia and Restic are supported for backup and restore operations. There is currently no workaround for this issue.
```
